python/python/raphtory/__init__.pyi

```diff
@@ -1260,23 +1347,15 @@ class Graph(GraphView):
         Returns:
             None:
         """
+
     def serialise(self) -> bytes:
         """
          Serialise Graph to bytes.
 
         Returns:
           bytes:
         """
```

Comment:
tidy activates the private storage by default, we might need to rejig this slightly

Reply:
I didn't do tidy - I just run stubs and fmt, but indeed it would be nice to have a OS version of tidy